src/posts/2020-10-02-introduction-to-WAI.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Introduction to Web Accessibility Initiative.
+description: Key facts about web accessibility initiative, what it is, and why it exists.
+category: Backgrounds
+author: Hamsa Harcourt
+further_reading:
+  - url: https://www.w3.org/WAI/about/ 
+    title:  About W3C WAI
+    source: W3C Web Accessibility Initiative
+  - url:  https://www.washington.edu/doit/what-web-accessibility-initiative-wai
+    title: "What is Web Accessibility Initiative (WAI)?"
+    source: "DO-IT - Disabilities, Opportunities, Internetworking, and Technology"
+  - url: https://en.wikipedia.org/wiki/Web_Accessibility_Initiative
+    title: Web Accessibility Initiative
+    source: Wikipedia
+date: 2020-10-02
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - background
+---
+
+The advent of the web brought infinite possibilities. It changed the way we learn, work and socialize because we were no longer limited by borders; real or imagined. However there was a problem: Lack of standard guidelines and proper structure. Due to the divergence of individual initiatives the [Web Accessibility Initiative (WAI)](https://www.w3.org/WAI/) was created.
```

Comment:
```suggestion
The advent of the web brought infinite possibilities. It changed the way we learn, work and socialize because we were no longer limited by borders; real or imagined. However there was a problem: Lack of standard guidelines and proper structure. Due to the divergence of individual initiatives the [Web Accessibility Initiative (<abbr>WAI</abbr>)](https://www.w3.org/WAI/) was created.
```

This will help codify this acronym for later uses. [Here's a little more info](https://developer.paciellogroup.com/blog/2019/03/short-note-the-abbreviation-appreciation-society/), if you're interested.